public/index.html

```diff
@@ -15,7 +15,7 @@ <h1 class='title'>The VandyHacks: Retro Edition Hacker Guide</h1>
     <script>
         window.onload = () => {
             if(window.location.hash) {
-                window.location.hash.scrollIntoView();
+                document.getElementsByName(window.location.hash.replace("#", ""))[0].scrollIntoView();
```

Comment:
Nice! Why not just getElementByName though?

Reply:
because that's not a thing?

Reply:
lmao thanks

Reply:
https://stackoverflow.com/a/7816899/7729429